lib/features/auth/repository/auth_repository.dart

```diff
@@ -34,6 +34,13 @@ class AuthRepository {
   }
 
   Future<void> signOut() async {
+    await _appAuth.endSession(
+      EndSessionRequest(
+        idTokenHint: await _secureStorage.read(key: SecureStorageKeys.idToken),
+        postLogoutRedirectUrl: 'com.example.grueneapp://oauthredirect',
```

Comment:
I am just talking about the dart code, not any environment variables. I think it would be nice to only have to have/adjust it in one place. I think my suggestion does not any environment variables or am I missing something?

Reply:
But if you change this variable, you have to change a value on some other places (android manifest etc.).
I think if you have a config variable but it does not have impact on all needed code fragments in the app, it's confusing.
But if you want I can change it, doesn't matter for me.

Reply:
Personally, I think its very much expected that config files (such as AndroidManifest.xml) have to be changed manifest. Accordingly I'd prefer it to be changed such that it is reusable in the dart code. This makes it less likely that single occurrences are missed or incorrect. Thanks!

Reply:
I updated the code.